buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -135,7 +108,7 @@ def collide_circle(left: Sprite, right: Sprite) -> bool: ...
 
 class collide_circle_ratio:
     ratio: float
-    def __init__(self, ratio: float): ...
+    def __init__(self, ratio: float) -> None: ...
```

Comment:
Is this correct for an init returning None?

Reply:
Yes, PEP 484